src/side-navigation/side-navigation.scss

```diff
@@ -1,205 +1,144 @@
-    }
-  }
-
-  > .iui-sidenav-content {
+  > .iui-content {
     display: flex;
     flex-direction: row;
-    height: calc(100% - #{$iui-l});
+    height: calc(100% - #{$iui-line-height});
 
-    > .iui-sidenav-items-container {
+    > .iui-content-container {
       display: flex;
       flex-direction: column;
+      width: 100%;
 
-      > .iui-sidenav-items {
+      > .iui-top {
         flex-grow: 1;
         display: flex;
         flex-direction: column;
-      }
 
-      > .iui-sidenav-items > .iui-sidenav-item-container,
-      > .iui-bottom-container > .iui-sidenav-item-container {
-        margin-bottom: -1px;
-        @include iui-sidenav-item-container;
+        > .iui-button:not(:last-of-type) {
+          border-bottom: none;
+        }
+      }
+      
+      > .iui-bottom > .iui-button {
+        border-bottom: none;
       }
     }
   }
 
   &.iui-collapsed {
-    min-width: $iui-l * 2;
-    max-width: $iui-l * 2;
+    min-width: $iui-icons-large * 3;
+    max-width: $iui-icons-large * 3;
   }
 
   &.iui-expanded,
-  > .iui-sidenav-content > .iui-sidenav-items-container {
+  > .iui-content > .iui-content-container {
     max-width: $iui-3xl * 3;
   }
 
-  &.iui-expanded > .iui-expand-container > .iui-icon {
+  &.iui-expanded > .iui-expand > .iui-icon {
     transform: rotate(180deg);
-  }
-}
-
-@mixin iui-sidenav-item-container {
-  display: flex;
-  flex-direction: row;
-  user-select: none;
-  height: $iui-l * 2;
-  cursor: pointer;
-  align-items: center;
-  transition: box-shadow $iui-speed-fast ease;
-  text-decoration: none;
-  box-sizing: border-box;
-  border-right: none;
-  border-left: none;
-
-  @include themed {
-    border-top: 1px solid t(iui-color-background-5);
-    border-bottom: 1px solid t(iui-color-background-5);
-    background-color: t(iui-color-background-1);
-  }
-
-  &:not(:last-child) {
-    border-bottom: none;
+    // NOT WORKING
+    animation: 5s ease roll;
   }
 
-  > .iui-sidenav-item-icon {
-    padding: 0 $iui-icons-default;
-    flex-shrink: 0;
-    height: $iui-icons-default;
-    width: $iui-icons-default;
-    transition: fill $iui-speed-fast ease;
+  .iui-button {
+    border-radius: 0;
+    width: 100%;
+    border-left: none;
+    border-right: none;
+    overflow: hidden;
 
-    @include themed {
-      fill: t(iui-icons-color);
+    &,
+    &:hover {
+      @include themed {
+        border-color: t(iui-color-background-5);
+      }
     }
-  }
 
-  > .iui-sidenav-item-caption {
-    transition: color $iui-speed-fast ease;
-    overflow-wrap: break-word;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    padding-right: $iui-m;
-
-    @include themed {
-      color: t(iui-text-color);
+    > .iui-icon {
+      width: $iui-icons-large;
+      height: $iui-icons-large;
+      flex-shrink: 0;
+    }
+  
+    > .iui-label {
+      margin-left: $iui-icons-large !important;
     }
-  }
 
-  &.iui-disabled {
-    cursor: not-allowed;
+    &.iui-expand {
```

Comment:
For me it is hard to read like that. I would say to have button restyling that is applied to all buttons here. And move this outside and maybe target it with `.iui-expand.iui-button` and do the same with `&:not(.iui-expand)` -> `.iui-content-container .iui-button`

Reply:
Ok, I _think_ I changed it to a way that you suggested.  Please check out the change and let me know if this is better.